maxim_g/HW6/task1.py

```diff
@@ -0,0 +1,26 @@
+# Task1. In the range from 1 to 10 determine
+# • even numbers that are divisible by 2,
+# • odd numbers, which are divisible by 3,
+# • numbers that are not divisible by 2 and 3
+
```

Comment:
good job
another way
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')